src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User | null;
```

Comment:
Handle it on the upper component
```suggestion
  user: User;
```